qualtran/bloqs/qft/qft_text_book.py

```diff
@@ -12,21 +12,40 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 from functools import cached_property
-from typing import Iterator
+from typing import cast, Iterator, Set
 
 import attrs
 import cirq
 from numpy.typing import NDArray
 
-from qualtran import GateWithRegisters, QUInt, Signature
+from qualtran import bloq_example, BloqDocSpec, GateWithRegisters, QUInt, Signature
+from qualtran.bloqs.basic_gates.hadamard import Hadamard
+from qualtran.bloqs.basic_gates.swap import TwoBitSwap
 from qualtran.bloqs.rotations.phase_gradient import PhaseGradientUnitary
+from qualtran.resource_counting import BloqCountT, SympySymbolAllocator
 from qualtran.symbolics import SymbolicInt
+from qualtran.symbolics.types import is_symbolic
 
 
 @attrs.frozen
 class QFTTextBook(GateWithRegisters):
-    r"""Textbook version of QFT using $\frac{N (N + 1)}{2}$ controlled rotations.
+    r"""Standard Quantum Fourier Transform from Nielsen and Chuang
 
+    Performs the QFT on a register of `bitsize` qubits utilizing
+    `bitsize` Hadamards and `bitsize` * (`bitsize` - 1) / 2 controlled Z
+    rotations, along with a reversal of qubit ordering specified via
+    `with_reverse` which defaults to `True`. `bitsize` can be provided numerically or symbolically.
+    More specific QFT implementations can be found:
+    - `ApproximateQFT` does not apply as small phases as standard QFT and relies on a specified rotation accuracy cutoff.
```

Comment:
"as small phases as" sounds bit awkward. Maybe refactor to something like "ignores small angle rotations"

Reply:
fixed